ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcarousel/v1/productcarousel/README.md

```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright 2019 Adobe Systems Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Product Carousel (v1)
+
+Product Carousel component is a server-side component written in HTL, allowing to display a list of featured products in a carousel style.
+The product is retrieved from Magento via GraphQL. This component can be used on any experience page.
```

Comment:
I'd write: The products are retrieved ...

Reply:
Changed